### ISSUE TITLE
Large vs small joint hypermobilty

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7459]
**Preferred term label:**
Add these terms and rearrange subhierarchy

FKBP14-related Ehlers-Danlos
OMIM:614557
7/7 (large) 7/7 (small)
PMID: 22265013

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
See this classification summarized by radiopaedia
Large joints

    [shoulders](https://radiopaedia.org/articles/shoulder?lang=us)
    [elbows](https://radiopaedia.org/articles/elbow?lang=us)
    [hips](https://radiopaedia.org/articles/hip-joint-1?lang=us)
    [knees](https://radiopaedia.org/articles/knee-joint-1?lang=us)
    [ankles](https://radiopaedia.org/articles/ankle-joint-2?lang=us)

Small joints

    [metacarpophalangeal joints](https://radiopaedia.org/articles/metacarpophalangeal-joint?lang=us)
    [proximal interphalangeal joints](https://radiopaedia.org/articles/interphalangeal-joint-of-the-hand?lang=us)
    [second to fifth metatarsophalangeal joints](https://radiopaedia.org/articles/metatarsophalangeal-joint?lang=us)
    [thumb interphalangeal joints](https://radiopaedia.org/articles/missing?article%5Btitle%5D=thumb-interphalangeal-joint&lang=us)
    [wrists](https://radiopaedia.org/articles/wrist?lang=us)

**Comment by pnrobinson** [comment]
added
